app/javascript/components/Stats.tsx

```diff
@@ -42,17 +42,26 @@ export const Stats = ({
   }, [value]);
 
   return (
-    <section className={cx("stats", className)}>
-      <h2>
+    <section className={classNames(
+      "text-4xl leading-tight p-8 border border-border rounded inline-grid content-between gap-2 bg-background text-foreground",
```

Comment:
Don't think this needs to be an inline-grid, can it just be grid?

Also, do we need `text-foreground`? It should be the default